tools/common/Application.cs

```diff
@@ -467,16 +467,19 @@ public void RunRegistrar ()
 			if (Registrar != RegistrarMode.Static)
 				throw new PlatformException (67, "Invalid registrar: {0}", Registrar); // this is only called during our own build
 
-			if (RootAssemblies.Count != 1)
-				throw ErrorHelper.CreateError (8, "You should provide one root assembly only, found {0} assemblies: '{1}'", RootAssemblies.Count, string.Join ("', '", RootAssemblies.ToArray ()));
+			if (RootAssemblies.Count < 1)
+				throw ErrorHelper.CreateError (129, "No assemblies found. You should provide at least one assembly.");
```

Comment:
the message should still mention _root_ assemblies since this is how it's currently documented

Reply:
👍